Gemfile

```diff
@@ -33,6 +33,8 @@ gem 'jbuilder'
 # Causes rails console to open pry [https://github.com/pry/pry-rails]
 gem 'pry-rails'
 
+gem 'jquery-rails'
```

Comment:
- [ ] Kindly add `gem 'ffi'` to your Gemfile so that users can successfully run your tests with no issues.